examples/diffusion/python_stable_diffusion_xl/README.md

```diff
@@ -18,9 +18,7 @@ Install dependencies
 
 ```bash
 pip install -U pip
-pip install -r requirements.txt
-# hip-python needs to be installed separately to not conflict with test packages
-pip install -i https://test.pypi.org/simple hip-python
+pip install -r torch_requirements.txt -r requirements.txt
```

Comment:
Same thing as above. if possible, leave this as pip install -r requirements.txt

Reply:
This doesn't work, it exhibits the same behavior with conflicting dependencies.